internal/driver/driver.go

```diff
@@ -699,3 +705,15 @@ func getUSBDeviceIdInfo(path string) (cardName string, serialNumber string, err
 	}
 	return
 }
+
+func (d *Driver) ValidateDevice(device models.Device) error {
+	_, err := d.getProtocolProperty(device.Protocols, UsbProtocol, AutoStreaming)
```

Comment:
is there a default value for AutoStreaming?

Reply:
Yes, the default value is false. 
Removed the `AutoStreaming` checking.